pymilo/transporters/baseloss_transporter.py

```diff
@@ -37,7 +37,7 @@
         # there is not glm models.
         legacy_version
 
-print("LEGACY-VERSION: ",legacy_version)
+# print("LEGACY-VERSION: ",legacy_version)
```

Comment:
I'm not sure if this comment is necessary. If it's useless, please remove it, otherwise disregard my comment.

Reply:
Yeah yeah I know, I have many Todos in the code which I'm going to do after the main tasksets for version .0.1, don't worry I will refactor them all but not in this PR.
a Fat Thanks.